test/cql-pytest/test_keyspace.py

```diff
@@ -207,7 +207,7 @@ def creates(count):
 def test_storage_options_local(cql, scylla_only):
     ksdef = "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : '1' } " \
             "AND STORAGE = { 'type' : 'LOCAL' }"
-    with new_test_keyspace(cql, ksdef) as keyspace:
+    with new_test_keyspace(cql, ksdef, force_no_tablets=True) as keyspace:
```

Comment:
This is sad, but not your fault. This test is faulty, it assumes that the existance of scylla_keyspaces is what determines if there's local storage or not, which is wrong.
I admit that making it not use tablets is easier than fixing this test.
We'll probably run into this problematic test again in the future, when we add more stuff into scylla_keyspaces, and will then need to fix this test for real :-(

Reply:
Fixing this is so simple, I might just do it in v2, instead of using this marker here and explaining why I did it in the patch notes.

Reply:
Fixed in v2 (properly).